From 04425f50fda09f61db8ede84fe177d8641fe8391 Mon Sep 17 00:00:00 2001
From: Paul Handrigan <Paul.Handrigan@cirrus.com>
Date: Wed, 4 Nov 2015 18:25:14 -0600
Subject: [PATCH 500/959] ASoC: cs35l34: Change method of getting codec pointer
 for 3.10

Change the way we get the codec pointer to be compatble with
kernel 3.10.

Change-Id: I1f6ba244afbb73392e9a5b132c519dcb11e0efdc
Signed-off-by: Paul Handrigan <Paul.Handrigan@cirrus.com>
Reviewed-on: http://gerrit.mot.com/801694
SME-Granted: SME Approvals Granted
Submit-Approved: Jira Key <jirakey@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Reviewed-by: Igor Kovalenko <igork@motorola.com>
---
 sound/soc/codecs/cs35l34.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/codecs/cs35l34.c b/sound/soc/codecs/cs35l34.c
index 36bdf86d97e..d19b467400f 100644
--- a/sound/soc/codecs/cs35l34.c
+++ b/sound/soc/codecs/cs35l34.c
@@ -243,7 +243,7 @@ static bool cs35l34_precious_register(struct device *dev, unsigned int reg)
 static int cs35l34_sdin_event(struct snd_soc_dapm_widget *w,
 		struct snd_kcontrol *kcontrol, int event)
 {
-	struct snd_soc_codec *codec = snd_soc_dapm_to_codec(w->dapm);
+	struct snd_soc_codec *codec = w->codec;
 	struct cs35l34_private *priv = snd_soc_codec_get_drvdata(codec);
 	int ret;
 
@@ -270,7 +270,7 @@ static int cs35l34_sdin_event(struct snd_soc_dapm_widget *w,
 static int cs35l34_main_amp_event(struct snd_soc_dapm_widget *w,
 		struct snd_kcontrol *kcontrol, int event)
 {
-	struct snd_soc_codec *codec = snd_soc_dapm_to_codec(w->dapm);
+	struct snd_soc_codec *codec = w->codec;
 	struct cs35l34_private *priv = snd_soc_codec_get_drvdata(codec);
 
 	switch (event) {
-- 
2.11.0

